### PR TITLE
Auto assign

### PR DESCRIPTION
**Pull request by changchanghwang**
### Description of change
- auto assign when pull request opend

### Pull-Request Checklist

- [ ] Code is up-to-date with the `main` branch
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [ ] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/JunbaeJs/junbae-mode/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by myunggyuns — APPROVED** [review]

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# A list of reviewers to be added to pull requests (GitHub user name)
+reviewers:
+  - myunggyuns
+  - sunnieeeYoon
+  - changchanghwang
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 2
```
Comment (sunnieeeYoon) [inline .github/auto_assign.yml:15]: Why is it 2 instead of 0?

File: .github/auto_assign.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# A list of assignees, overrides reviewers if set
+assignees:
+  - changchanghwang
+
+# A number of assignees to add to the pull request
+# Set to 0 to add all of the assignees.
+# Uses numberOfReviewers if unset.
+numberOfAssignees: 1
```
Comment (sunnieeeYoon) [inline .github/auto_assign.yml:24]: why is it 1?

File: .github/auto_assign.yml (hunk 1 above)
Comment (changchanghwang) [reply .github/auto_assign.yml:15]: Even if the number of contributors increases, only 2 people review.

File: .github/auto_assign.yml (hunk 2 above)
Comment (changchanghwang) [reply .github/auto_assign.yml:24]: I think assignee is charge of pr. so It should be one. if you want, I add you in list

**Review by sunnieeeYoon — APPROVED** [review]

**Pull request merged**